syntax = "proto3";

package route;

// a simple bidirectional streaming example
// client发送文件存储的iniLoc给server，server返回一个ok
// server根据iniLoc完成二维拼接任务，发送finLoc给client，client回复ok

service localGuide {
    rpc GetLocation(stream IniLoc) returns (stream FinLoc) {}
}

message IniLoc {
    string ini_location=1;
}

message FinLoc {
    string fin_location=1;
}

